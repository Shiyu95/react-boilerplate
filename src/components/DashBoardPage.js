import React from 'react';


const DashBoardPage =()=>(
    <div>
      DashBoard page content
    </div>
  );

  export default DashBoardPage;