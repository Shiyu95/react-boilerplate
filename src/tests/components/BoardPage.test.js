import React from 'react';
import {shallow} from 'enzyme';
import DashBoardPage from '../../components/DashBoardPage';


//test first one 
test('should render DashBoardPage', () => {
    const wrapper = shallow(<DashBoardPage  />);
    expect(wrapper).toMatchSnapshot();

});