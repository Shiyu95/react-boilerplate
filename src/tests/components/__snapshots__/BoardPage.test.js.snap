// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashBoardPage 1`] = `
<div>
  DashBoard page content
</div>
`;
